CHANGELOG.md

```diff
@@ -1,6 +1,69 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.12.3:
+	- - fixup FR push_over_rsync for FR i2 (ENT-4587)
+	- Added AIX support to body perms system_owned (ENT-4773)
+	- Added ability to avoid limiting robot agents (CFE-3161)
+	- Added and transitioned to using master_software_updates shortcut
+	  (ENT-4953)
+	- Added continual checking for policy_server state (CFE-3073)
+	- Added documentation how to enable systemd unit management and disable agents on all hosts
+	  (CFE-3416)
+	- Added package_module for snap (CFE-2811)
+	- Added scripts and templates for Federated Reporting (ENT-4473)
+	- Added support for 'awk' filters in the FR dump-import process
+	  Ticket: (ENT-4839)
+	- Added support for configuring abortclasses and abortbundleclasses via augments
+	  (ENT-4823)
+	- Added support for filtering in both dump and import
+	  phases of the FR ETL process (ENT-4839)
+	- Added support for ordering FR awk and sed scripts (ENT-4839)
+	- Added support for setting periodic package inventory refresh interval via augments
+	  (CFE-2771)
+	- Always set files_single_copy from augments if available (CFE-3064)
+	- Changed FR policy to honor target_state properly (ENT-4874)
+	- Copy .awk and .sed files from masterfiles to inputs (ENT-4839)
+	- Do not run DB maintenance tasks on a passive HA hub (ENT-4706)
+	- Enabled systemd unit management by default (CFE-2429)
+	- Fixed agent disabling on systemd systems (CFE-2429)
+	- Fixed agent disabling on systemd systems (CFE-3416)
+	- Fixed cleanup of future timestamps from status table
+	  (ENT-4331, ENT-4992)
+	- Fixed pkgsrc in case where multiple Prefix paths are returned for pkg_install
+	  (CFE-3153)
+	- Fixed pkgsrc module on Solaris/NetBSD (CFE-3151)
+	- Fixed re-spawning of cf-execd or cf-monitord after remediating duplicate concurrent processes
+	  (CFE-3150)
+	- Fixed state ownership on aix (ENT-4773)
+	- Fixed synchronization of important configuration files from active to passive hub
+	  (ENT-4944)
+	- Fixed the CFEngine 3.7.x class guard in standalone_self_upgrade.cf
+	  (CFE-3182)
+	- Made keys of all types from feeder hubs trusted on a superhub
+	  (ENT-4917)
+	- Replace 64-bit arch with actual one (ENT-5005)
```

Comment:
I would redact this or re-write the line to be more descriptive. It seems unrelated to the product and more about our internal CI stuff.